src/components/clickable-card/ClickableCard.styles.ts

```diff
@@ -4,23 +4,25 @@ export const styles = {
     maxWidth: '62px',
     maxHeight: '62px'
   },
-  titleWithDescription: {
-    minWidth: '110px',
-    margin: 0,
-    mb: 0,
-    lineHeight: '24px',
-    textAlign: 'start'
-  },
-  title: {
-    whiteSpace: 'nowrap',
-    overflow: 'hidden',
-    textOverflow: 'ellipsis',
-    color: 'basic.black',
-    typography: { xs: 'h6' },
-    m: 0
-  },
-  description: {
-    color: 'primary.500',
-    typography: { xs: 'body2' }
+  titleComp: {
```

Comment:
I see no reason to change titleWithDescription titleComp, for me it is more unclear